pkg/devenvironment/name.go

```diff
@@ -26,6 +26,11 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+const (
+	// defaultOktetoFilename is the default name of the manifest file
+	defaultOktetoFilename = "okteto.yml"
```

Comment:
what if is `okteto.yaml` how did we determine that this is the default one?

Reply:
IIRC, we handle `okteto.yaml` and `okteto.yml` in the same way, so there is no actually a single default one. Also, this won't work when the "default" manifest is `.okteto/okteto.yml`,  `.okteto/okteto.yaml` or a docker-compose instead of an Okteto manifest.

The main problem of this issues is that when we deploy something specifying the manifest path, and that is one of the ones we automatically infer, when we execute `okteto up` without specifying the path, it doesn't get the name it should.

For example, you can still reproduce the issue with this repository: https://github.com/ifbyol/go-getting-started and the branch `ifbyol/move-default-manifest`

Reply:
We decided to go with `okteto.yml` since it's the default one we use for `okteto up` command

Reply:
Reopening discussion:

After speaking with @ifbyol he suggested to go with all the possible okteto manifest paths that the `okteto up` command can infer. I've gone ahead and added the support for all the possible files, so if the user set one of them in the configmap such as `.okteto/okteto.yaml` whenever he runs `okteto up` he'll get the correct dev environment. 

cc: @pchico83 since this new solution is a little more complex than the original request. Do you agree with the proposed solution?

Reply:
@jLopezbarb sounds good